assets/src/edit-story/components/panels/textStyle/useRichTextFormatting.js

```diff
@@ -112,19 +114,24 @@ function useRichTextFormatting(selectedElements, pushUpdate) {
         handleClickBold: () => selectionActions.toggleBoldInSelection(),
         // All these keep their arguments:
         handleSelectFontWeight: selectionActions.setFontWeightInSelection,
+        handleResetFontWeight: (weight) => {
+          push(htmlFormatters.setFontWeight, weight);
+        },
         handleClickItalic: selectionActions.toggleItalicInSelection,
         handleClickUnderline: selectionActions.toggleUnderlineInSelection,
         handleSetLetterSpacing: selectionActions.setLetterSpacingInSelection,
         handleSetColor: selectionActions.setColorInSelection,
       };
     }
 
-    const htmlFormatters = getHTMLFormatters();
-
     return {
       handleClickBold: (flag) => push(htmlFormatters.toggleBold, flag),
-      handleSelectFontWeight: (weight) =>
-        push(htmlFormatters.setFontWeight, weight),
+      handleSelectFontWeight: (weight) => {
+        push(htmlFormatters.setFontWeight, weight);
+      },
+      handleResetFontWeight: (weight) => {
+        push(htmlFormatters.setFontWeight, weight);
+      },
```

Comment:
These appear to be the same function? 😜

Reply:
yeah, i was curious what you'd think on this! i'll clean it up to share the same function